src/FormFileInput.tsx

```diff
@@ -15,6 +15,7 @@ export interface FormFileInputProps extends BsPrefixProps, BsCustomPrefixProps {
   isValid?: boolean;
   isInvalid?: boolean;
   lang?: string;
+  onChange?: (event: any) => void;
```

Comment:
Better to extend from `InputHTMLAttributes` so we don't have to manually type onChange.